docs/DeveloperGuide.adoc

```diff
@@ -153,6 +153,108 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+// tag::loans[]
+=== Loans Feature
+==== Implementation
+
+The Loans feature exists outside of the Account/Transaction mechanisms.
+It adds a separate `LoansManager` alongside the main `AccountsManager`, with `Loan` objects stored internally in an `internalList`.
+
+The following class diagram demonstrates the association between the `LoansManager` and `Loan` objects
+(miscellaneous methods such as getters and setters are omitted):
+
+image::LoanClassDiagram.png[]
+
+Each `Loan` object has the following attributes:
+
```

Comment:
Great explanation of class attributes. To improve and reduce ambiguity, you could provide some examples for the direction attribute.